app/src/main/res/layout/activity_select_details.xml

```diff
@@ -0,0 +1,310 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Please try to use constraint layout as much as possible instead of the linear layout.